docs/resources/README.md

```diff
@@ -0,0 +1 @@
+This directory contains resources (images) used in docs
```

Comment:
Is this file really needed? I think this directory name is self-explanatory :)

Reply:
Github dont allow to create empty directory from web interface

Reply:
But it's no longer an empty directory, so maybe it could be removed now?

Reply:
Ok!

Reply:
Done!

Reply:
@nevack, solved?